vtu11/impl/vtu11_impl.hpp

```diff
@@ -146,6 +150,43 @@ void write( const std::string& filename,
   output.close( );
 }
 
+//ParallelWrite generates a pvtu file and accordingly the vtu pieces in a subfolder
+//Each piece consists of a set of points and cells
+template<typename MeshGenerator, typename Writer = AsciiWriter>
+void parallelWrite( const std::string& path,
+                    std::string baseName,
```

Comment:
```suggestion
                    const std::string& baseName,
```
same comment as above